requirements/bddtests.in

```diff
@@ -6,4 +6,5 @@ factory-boy
 h-matchers
 webtest
 behave
+filelock
```

Comment:
We're not running the bddtests in parallel but they shared the `db_engine()` fixture which now uses `filelock`, so they need this dependency.